src/assignments/2-operator.js

```diff
@@ -14,6 +14,40 @@ output example
 */
 // start here writing the code for program1
 
+const num1 = Number(prompt("Number 1: "));
+const num2 = Number(prompt("Number 2: "));
+
+const add = num1 + num2;
+console.log(`${num1}+${num2}=${add}`);
+
+const sub = num1 - num2;
+console.log(`${num1}-${num2}=${sub}`);
+
+const multiply = num1 * num2;
+console.log(`${num1}*${num2}=${multiply}`);
+
+const divide = (num1, num2) => {
+	if (num2 == 0) {
+		console.log("undefined");
+	} else {
+		let answer = num1/num2.toFixed(2);
```

Comment:
good job! always check the num2 whether bigger than 0 or not before doing divide calculation!